deploy/helm/quarks/templates/service-account-hooks.yml

```diff
@@ -13,4 +13,17 @@ metadata:
     "helm.sh/hook": {{$hook}}
     "helm.sh/hook-weight": "-2"
     "helm.sh/hook-delete-policy": before-hook-creation
+{{- if $.Values.global.image.credentials }}
+imagePullSecrets:
+- name: {{$hook}}-pull-secret
```

Comment:
I don't think you really need to define additional secrets, you can just reference the operator service account secret:

```
- name: {{ template "cf-operator.serviceAccountName" . }}-pull-secret
```

I don't care too much, but generally dislike redundant config. Just makes debugging harder than it needs to be. :)